demisto_sdk/commands/common/update_id_set.py

```diff
@@ -693,6 +693,34 @@ def create_common_entity_data(path, name, to_version, from_version, pack):
     return data
 
 
+def get_pack_metadata_data(file_path, print_logs: bool):
+    try:
+        if print_logs:
+            print(f'adding {file_path} to id_set')
+
+        json_data = get_json(file_path)
+        pack_data = {
+            "name": json_data.get('name'),
+            "current_version": json_data.get('currentVersion'),
+            "author": json_data.get('author', ''),
+            'certification': 'certified' if json_data.get('support', '').lower() in ['xsoar', 'partner'] else '',
+            "tags": json_data.get('tags', []),
+            "use_cases": json_data.get('useCases', []),
+            "categories": json_data.get('categories', [])
+        }
+
+        pack_id = get_pack_name(file_path)
+        if pack_id:
+            pack_data['id'] = pack_id
+
+        pack_name = pack_data['name']
+        return {pack_name: pack_data}
+
+    except Exception as exp:  # noqa
+        print_error(f'failed to process {file_path}, Error: {str(exp)}')
```

Comment:
```suggestion
        print_error(f'Failed to process {file_path}, Error: {str(exp)}')
```

also might be a good idea to add traceback here.

Reply:
done